tests/zfs-tests/tests/functional/xattr/cleanup.ksh

```diff
@@ -30,6 +30,9 @@
 . $STF_SUITE/include/libtest.shlib
 . $STF_SUITE/tests/functional/xattr/xattr_common.kshlib
 
+del_user $ZFS_USER
+del_group $ZFS_GROUP
```

Comment:
This is already done a few lines down.  Did it need to be moved up for cleanup, and if so can we remove the other calls below.

Reply:
No idea where that came from :scissors: